packages/tradershub/src/features/cfd/modals/MT5PasswordModal/ButtonGroups/CreateAccountButton.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I think this component can be reused. Right?
If yes then improve it in other PR for Dxtrade.
Otherwise ignore this comment.

Reply:
yes, that was the intention in the first place